app/controllers/mission_control/jobs/many_retries_controller.rb

```diff
@@ -0,0 +1,15 @@
+class MissionControl::Jobs::ManyRetriesController < MissionControl::Jobs::ApplicationController
+  def create
+    retried_jobs = job_ids.map do |job_id|
+      job = ActiveJob.jobs.find_by_id(job_id)
+      job.retry if job&.failed?
+    end.compact
```

Comment:
Same comment as above. But we should be able to call the `.retry_all` on the collection. (See: MissionControl::Jobs::BulkRetriesController for an example).